apps/web/src/components/monitoring/SentryErrorBoundary.tsx

```
@@ -0,0 +1,86 @@
+          <h1 className="text-xl font-semibold text-gray-900 mb-2">
+            Algo deu errado
+          </h1>
+          <p className="text-gray-600 mb-6">
+            Ocorreu um erro inesperado. Nossa equipe foi notificada automaticamente.
+          </p>
+          
+          <div className="space-y-3">
+            <button
+              onClick={resetError}
+              className="w-full bg-blue-600 text-white px-4 py-2 rounded-md hover:bg-blue-700 transition-colors"
+            >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)